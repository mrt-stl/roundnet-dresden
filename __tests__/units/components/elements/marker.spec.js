import { create } from "react-test-renderer"
import Marker from "../../../../components/elements/marker"

/**
 * This is a snapshot test. A snapshot test is taking when the component
 * doesn't change that often. Avoid snapshot testing if the component is
 * about to change often. Marker component should not change this often.
 */

describe("Marker component", () => {
    test("Matches the snapshot", () => {
        const marker = create(<Marker />)
        expect(marker.toJSON()).toMatchSnapshot()
    })
})