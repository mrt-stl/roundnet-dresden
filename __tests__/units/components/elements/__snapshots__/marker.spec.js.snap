// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Marker component Matches the snapshot 1`] = `
<div
  className="jsx-1110152491 marker-container"
>
  <img
    className="jsx-1110152491"
    src="/static/icon/marker.svg"
  />
</div>
`;
